tpm2/tpm2.go

```diff
@@ -637,7 +681,23 @@ func decodeUnseal(in []byte) ([]byte, error) {
 
 // Unseal returns the data for a loaded sealed object.
 func Unseal(rw io.ReadWriter, itemHandle tpmutil.Handle, password string) ([]byte, error) {
```

Comment:
Make this a wrapper around `UnsealWithSession`

Reply:
done